### PR TITLE
All chart stuff

### LINKED ISSUE #720: Charting - Design Guidelines Part 1
Add design guidelines for usage of various charts and graphs to represent data.

### PR DESCRIPTION
**Pull request by gdoyle1**
Close #720 #747 

Added the chart color system to the styles section and added charting design guidelines

### CODE REVIEW

**Comment by gdoyle1** [comment]
@mcarrano Changes have been made! I spoke with Stacy and she said H2/H3s are fine as they are now. Let me know if you need me to do anything else

File: packages/patternfly-4/content/design-guidelines/usage-and-behavior/Charts for colors/design.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+path: "/design-guidelines/usage-and-behavior/charting"
+
+# Charts for colors
```
Comment (mcarrano) [inline packages/patternfly-4/content/design-guidelines/usage-and-behavior/Charts for colors/design.md]: I think you mean "Colors for charts"? :)

**Comment by gdoyle1** [comment]
@mcarrano Just fixed the typos. I renamed the usage-and-behavior guidelines section to "charts" and then added everything back into that folder (.md and imgs folder). I added "charts" to the usage-and-behavior part of the navigation.

**Comment by gdoyle1** [comment]
@mcarrano Okay this should now be ready to go!

File: packages/patternfly-4/content/design-guidelines/styles/charting_color_system.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+path: /design-guidelines/styles/charting-color-system
```
Comment (mindreeper2420) [inline packages/patternfly-4/content/design-guidelines/styles/charting_color_system.md]: @gdoyle1 the path definition needs to be bracketed with `---` at top and bottom. Currently, the site cannot locate this page. You also need quotations around the path.

```
---
path: "/design-guidelines/styles/charting-color-system"
---
```

File: packages/patternfly-4/content/design-guidelines/styles/colors-for-charts.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+path: /design-guidelines/styles/colors-for-charts
```
Comment (mindreeper2420) [inline packages/patternfly-4/content/design-guidelines/styles/colors-for-charts.md]: same thing here:

```
---
path: "/design-guidelines/styles/colors-for-charts"
---
```

File: packages/patternfly-4/content/design-guidelines/styles/charting_color_system.md (hunk 2 above)
Comment (gdoyle1) [reply packages/patternfly-4/content/design-guidelines/styles/charting_color_system.md]: how do I delete this? It's only supposed to be a path for "charts" in design-guidelines and then "colors-for-charts" in styles

File: packages/patternfly-4/content/design-guidelines/styles/colors-for-charts.md [hunk 4]
```diff
@@ -1,4 +1,6 @@
-path: /design-guidelines/styles/colors-for-charts
+---
+path: ""/design-guidelines/styles/colors-for-charts"
```
Comment (mindreeper2420) [inline packages/patternfly-4/content/design-guidelines/styles/colors-for-charts.md]: @gdoyle1  The extra `"` at the beginning of the path needs to be removed.

File: packages/patternfly-4/content/design-guidelines/styles/colors-for-charts.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
+---
```
Comment (mindreeper2420) [inline packages/patternfly-4/content/design-guidelines/styles/colors-for-charts.md]: @gdoyle1 Here is the block of code to use at the top of the page for referencing the GIFs:

```
---
path: "/design-guidelines/styles/colors-for-charts"
---
import theSystem from './thesystem.gif';
import oneFamily from './onefamily.gif';
```

File: packages/patternfly-4/content/design-guidelines/styles/colors-for-charts.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+## Chart colors
+
+Colors for charts includes a recommended set of colors proposed for bar charts, donut charts and pie charts. There are six base colors that the system is built on. Each base color has its own color family, which includes lighter and darker hues of the base. These families use a monochromatic color order. They are organized in a way that increases contrast while still looking extremely visually appealing. The variation of colors will depend on the type and amount of data at hand.
+
+![Chart colors](chartcolors.png)
+
+## The system
+Although the system may seem complex, it is completely customizable. The idea is that you can put together your desired colors and amount of colors and the system will still sort them correctly. Here’s how the system works:
+
+![The system gif](thesystem.gif)
```
Comment (mindreeper2420) [inline packages/patternfly-4/content/design-guidelines/styles/colors-for-charts.md]: Change this to

```
<img src={theSystem} />
```

File: packages/patternfly-4/content/design-guidelines/styles/colors-for-charts.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+If the set of data includes more than six variables, we recommend applying the multichromatic order system, which uses all of the colors in the color system, to the data. This includes a mix of base colors and colors from their family. An example of this type of data would be the cost of living expenses during a period of time.
+
+![Chart with many variables](manyvariables.png)
+
+## Using the system with one color family
+
+You can also use the color order system with one color family. This creates a monochromatic effect. It works using the same logic and order. We recommend using this system for charts with nested variables. The order begins by going through each base color, then lightest, darkest, second lightest, and ending with second darkest. In variable-terms: #1 [color]-300 (base color), #2 [color]-100, #3 [color]-500, #4 [color]-200, #5 [color]-400. The hues will always follow this order.
+
+![One color family gif](onefamily.gif)
```
Comment (mindreeper2420) [inline packages/patternfly-4/content/design-guidelines/styles/colors-for-charts.md]: Change this to:

```
<img src={oneFamily} />
```

**Pull request merged**